### PR TITLE
import fix

### PR DESCRIPTION
**Pull request by sanjayuconn**
# Peer Review Information

- Removed the synthetic import in init and fixed corresponding imports

# Pull Request Check List

- [ ] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

### CODE REVIEW

File: notebooks/SynthDet_bounding_box_visualization.ipynb [hunk 1]
```diff
@@ -21,7 +21,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
```
Comment (BlairLee) [inline notebooks/SynthDet_bounding_box_visualization.ipynb:24]: Please clear up all the outputs in the notebook.

File: datasetinsights/datasets/__init__.py [hunk 2]
```diff
@@ -12,6 +11,4 @@
     "GroceriesReal",
     "GoogleGroceriesReal",
     "NyuDepth",
-    "SynDetection2D",
```
Comment (BlairLee) [inline datasetinsights/datasets/__init__.py:15]: [mustfix] If we remove `SynDetection2D`, it cannot find `SynDetection2D` class in `Dataset.find()`.

Comment (sanjayuconn) [reply datasetinsights/datasets/__init__.py:15]: Yes I agree. We need to move usim.py under the datasets in this case, usim.py and synthetic.py have cyclic dependency.

**Comment by adason** [comment]
@sanjayuconn  We should remove `from datasetinsights.io.usim import Downloader, download_manifest` in `Synthetic.py` and remove calls that refer to download data for unity simulation. This code path is already dead after @Saurav-D makes `SynthDetection2D` to directly download from public http url.

**Comment by sanjayuconn** [comment]
Will send new PR.

**Pull request closed without merging**